ci/kind/test-e2e-kind.sh

```diff
@@ -141,6 +142,18 @@ case $key in
 esac
 done
 
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
maybe you could define this closer to the top and replace all `$(dirname $0)` occurrences with `$THIS_DIR`

Reply:
Updated. Thanks.